File: DSA/Abhyudaya_590025624/Day_2/Question4.py
n=int(input("Enter the number of elements in the array: "))
ar=[]
lar=0
print("Enter the elements in the array: ")  
for i in range(n):
    ar.append(int(input()))
    if (ar[i]>lar):
        lar=ar[i]
print("The largest element in the array is: ", lar) 
#~abhyudaya_590025624_DAY2 done

File: DSA/Aaditya_590024760/Day_2/Question4.c
#include<stdio.h>
int num[50];
int main(){
 int a;
 printf("Enter The number of elements in the array: " );
 scanf("%d",&a);

 printf("Enter %d Elements: ", a);
 for(int j = 0; j<a ; j++){
    scanf("%d", &num[j]);
 }
 int large = 0;
 for(int j= 0; j<a ;j++){
    if(num[j]>large){
        large = num[j];
    }
    else if(large > num[j]){
        large = large;
        }
    }
    printf("Largest Number: %d", large);
    return 0;
 }
